### ISSUE TITLE
Always opening 2 browser instances 

### ISSUE THREAD

**Issue by suresh-gurajala** [RankWeis/uppercut#42]
This is my feature file to open browser

```
Feature: Launch Chrome browser
  Background:
    * def osName = karate.os.name
    * def projectPath = karate.properties['user.dir']
    * def downloadPath = projectPath+"/" + downloadPath
    * if (osName.toLowerCase().includes('window')){downloadPath=downloadPath.replace("/null","").replaceAll("/","\\")}
    * def session = { capabilities: { alwaysMatch: { browserName: 'chrome', 'goog:chromeOptions': { args: [ '--window-size=1920,1080','--remote-allow-origins=*'] , 'prefs':{'plugins.always_open_pdf_externally':true, 'download.default_directory': '#(downloadPath)'} } } } }
    * def driver = { type: 'chromedriver',executable: '#(projectPath)/src/test/resources/drivers/chromedriver', webDriverSession: 'session'}
    * def linuxDriver = projectPath+'/src/test/resources/drivers/chromedriver'
    * def windowsDriver = projectPath+'/src/test/resources/drivers/chromedriver.exe'
    * if(osName.toLowerCase().includes('linux')){ driver.executable = linuxDriver ;driver.webDriverSession = session } else{driver.executable = windowsDriver;driver.webDriverSession = session}
    * configure driver = driver
    * configure continueOnStepFailure = true

  @LaunchBrowser
  Scenario: Launch Chrome browser
    Given driver baseUrl
    * driver.maximize()
    * driver.timeout(180000)
    * def driver = driver
    * print ' ========= Url launched in Chrome ========= '
    * print driver.getOptions()
```

**Comment by RankWeis** [comment]
Hi, how are you running this feature file? Is it through the gutter or through a run configuration? Does this not happen when executing through a command line?

**Comment by calvario-31** [comment]
Hi suresh, wold be good to upload a small project to reproduce

**Comment by RankWeis** [comment]
As best as I can tell, this might have been happening if you were running via tag, which would run both the files in your main folder and in `target/` - so effectively running the test twice. This should be resolved in 2.1.0, and as there has been no further update from the commenter, I have to use a best guess here and assume that was what was happening, so I am closing this now.

**Comment by RankWeis** [comment]
@suresh-gurajala I apologize for the delay in fixing this but I believe the newest version of uppercut in intellij should certainly address this issue, there were a few different instances of this happening that I believe I have resolved. See #108
